_data/contributors.yml

```diff
@@ -66,6 +66,11 @@
   role: PPMC, Committer
   org:
 
+- name: Haitao Liu
```

Comment:
Should we remove the extra space before "Flavio Paiva Junqueira" at line 57?

Reply:
There is exactly one space between all of the entries. I think it is correct?

Reply:
Yes, line 69 is right, but line 57 is wrong.

Reply:
Sorry I'm still not seeing what you are talking about, can you use the suggest an edit?

Reply:
Sorry, it seem that my browser render the layout a little bit wrong.